spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1693,7 +1693,10 @@ properties (with the prefix stripped) when the local `EntityManagerFactory` is c
 
 [[howto-configure-hibernate-naming-strategy]]
 === Configure Hibernate Naming Strategy
-Hibernate defines `Physical` and `Implicit` naming strategies. Spring Boot configures
+Since version 5, Hibernate defines `Physical` and `Implicit` naming strategies which you
+can configure in Spring Boot by providing a fully qualified class name to the properties
+`spring.jpa.hibernate.naming.physical-strategy` and
+`spring.jpa.hibernate.naming.implicit-strategy`. Spring Boot configures
```

Comment:
I feel stronger about this that the "since version 5"-thing above. Yes, the properties will be suggested by an IDE, but not everything takes place in an IDE (and not everyone knows or uses that feature in an IDE). 

When browsing the docs, I would have to read the section, infer that `spring.jpa.hibernate.naming.implicit-strategy` exists, and validate it by looking into the properties list in the appendix (which I'm not necessarily aware of).